python/qpid_dispatch/management/qdrouter.json

```diff
@@ -570,6 +570,11 @@
                     "graph": true,
                     "description": "The total number of settled deliveries that were held in the router for more than 10 seconds."
                 },
+                "deliveriesStuck": {
+                    "type": "integer",
+                    "graph": true,
+                    "description": "The current number of deliveries that are unsettled and have been held in the router for more than 10 seconds."
+                },
```

Comment:
Would you consider a 'peakDeliveriesStuck' stat? 
Picture a network where some clients go down and zillions of deliveries get stuck. Then those clients come back and the stuck number goes to zero. This would provide some indication of an issue even though the current state is fine.

Reply:
Do we need this?  INFO logs are issued when a link has stuck deliveries and the count is available through the prometheus-format scraper.
Since this is more of a gauge than a counter, having a peak value might be appropriate.  Would it also require a way to be reset?

Reply:
Live stats is fine. Adding more becomes a slippery slope towards too much.